src/ObjLoading/Game/T6/BSP/BSPUtil.cpp

```diff
@@ -0,0 +1,173 @@
+#pragma once
+
+#include "BSPUtil.h"
+
+#include <cmath>
+#include <format>
+
+namespace BSP
+{
+    std::string BSPUtil::getFileNameForBSPAsset(std::string& assetName)
+    {
+        return std::format("BSP/{}", assetName);
```

Comment:
Folders are generally lowercase. On Windows this does not make a difference, on Linux it does.